gradle-plugin/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'groovy'
+apply plugin: 'maven'
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+    compile 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+}
+
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
+
+group = 'io.realm'
+version = new File("${projectDir}/../version.txt").text.trim();
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            repository(url: "file://localhost${System.env.ANDROID_HOME}/extras/android/m2repository")
```

Comment:
AFAIK it is set, however there are some caveats. I had a problem in another project, and if you launch Android Studio from the launcher it doesn't inherit any environment variables in `.bash_profile`. So far the only work-around I found was a apple-script that opens Android Studio from a terminal :(